platform/node/index.d.ts

```diff
@@ -125,5 +130,90 @@ declare module '@maplibre/maplibre-gl-native' {
      * Call to permanently dispose the internal map resources, instance can't be used for further render calls
      */
     release: () => void;
+
+    /**
+     * Add source to map's style
+     */
+    addSource: (sourceId: string, source: {}) => void
+
+    /**
+     * Remove source from map's style
+     */
+    removeSource: (sourceId: string) => void
+
+    /**
+     * Add layer to map's style
+     */
+    addLayer: (layer: {}, beforeId?: string) => void
+
+    /**
+     * Remove layer from map's style
+     */
+    removeLayer: (layerId: string) => void
+
+    /**
+     * Add image to map's style
+     */
+    addImage: (imageId: string, image: any) => void
+
+    /**
+     * Remove image from map's style
+     */
+    removeImage: (imageId: string) => void
+
+    /**
+     * Set the extent of the zoom for a specified layer
+     */
+    setLayerZoomRange: (layerId: string, minzoom: number, maxzoom: number) => void;
+
+    /**
+     * Set the value for a layer's property
+     */
+    setLayoutProperty: (layerId: string, name: string, value: string) => void;
+
+    /**
+     * Set filter for specified style layer
+     */
+    setFilter: (layerId: string, filter: [] | null | undefined) => void;
+
+    /**
+     * Set the center of the map
+     */
+    setCenter: (center: any) => void;
```

Comment:
```suggestion
    setCenter: (center: [number, number]) => void;
```

https://github.com/maplibre/maplibre-gl-native/blob/02c196a4721ba0c0a72a60597854f2bf26fe64a7/render-test/parser.cpp#L768-L771